__init__.py

```diff
@@ -47,6 +52,13 @@ class SystemCommand(Enum):
 
 
 class DeviceControlCenterSkill(NeonSkill):
+    user_config_path = expanduser("~/.config/neon/neon.yaml")
```

Comment:
`ovos_config.locations.USER_CONFIG` should handle this path under OVOS too; also if `XDG_CONFIG_HOME` is overridden, i.e. Docker

Reply:
As it turns out this is not even used, not sure why it was here...maybe an early attempt at editing the configs before I knew about the messages? Regardless, I'm deleting it.